packages/backend/server/src/core/mail/job.ts

```diff
@@ -76,6 +75,7 @@ export class MailJob {
                 encoding: 'base64',
               },
             ];
+            workspaceProps.avatar = 'cid:workspaceAvatar';
```

Comment:
[![???](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/default-memes/questionmarks.gif)](https://app.graphite.dev//settings/meme-library?org=toeverything) damn it